src/geb/package.lisp

```diff
@@ -21,8 +21,12 @@
   (commutes          pax:function)
   (commutes-left     pax:function)
   (!->               pax:function)
-  (so-eval           pax:function)
-  (so-hom-obj        pax:function)
+  (so-eval           (pax:method () (<natobj>)))
+  (so-eval           (pax:method () (<substobj>)))
+  (so-eval           pax:generic-function)
```

Comment:
we should probably move this to the generic package we have for generic functions?